src/viper/pretty.ml

```diff
@@ -14,6 +14,11 @@ let semi ppf () = fprintf ppf ";@ "
 let pp_info ppf NoInfo = ()
 
 let rec pp_prog ppf p =
+  marks := p.at :: !marks;
```

Comment:
maybe back this out?
otherwise it will underline the whole program on error...
can we limit to the `actor { ... }`?

Reply:
Done in f4d82dc7e. We can bring it back any time if it appears to be useful.